examples/none.py

```diff
@@ -1,10 +1,11 @@
 """
 This driver does not do any action.
 """
-from rose.common import obstacles, actions
+from rose.common import actions
```

Comment:
Better silence the unused import with #NOQA. All driver module must import both modules.